Bixal/move.mil
# Feature/44/paragraph warning callout

Thanks for submitting a pull request! Below are a few things you can do to help us more quickly review your changes.

## Checklist

I have…

      against my changes

## Summary of Changes

This pull request…

- Implements the "Warning/Callout Box" paragraph type. Details in Issue #44 

### More details:

- The paragraph largely pulls from USWDS alert box styles, invoked through a 'usa-alert' class. 
- USWDS provides four types of alerts: info(blue), warning(yellow), error(red) and success(green).
- Those alert types are invoked through an option field, the value of which generates the necessary class in a new TWIG template for the paragraph.
- I reworked the options in the alert types field to align with these defaults, accounting for the config change included in this PR.

## Testing

To verify the changes proposed in this pull request…

1. Pull code 
1. run npm lint to check integrity of sass with content updates and changes to folder structure. _Note: I believe there will still be a few warnings._
1. run npm build to incorporate sass updates
1. Create four new paragraphs in your local environment, one for each alert type (with its associated Box Type option selected), to make sure it renders properly. You should get something like the screenshot below.

## Screenshots

From local environment:
![screen shot 2018-04-03 at 6 24 21 pm](https://user-images.githubusercontent.com/29130580/38321065-6e67d3ee-3804-11e8-8b5b-1efa7648f2fc.png)
